source/engagement/financial_detail/asset_list.rst

```diff
@@ -0,0 +1,80 @@
+``asset_type_id`` is an optional filter parameter that can be used to filter the asset type list. 
+
+``entity_id`` and ``service_point_id`` are optional filter parameters that can be used to filter the asset financial detail list. 
+The financial details of an asset type can be specified by entity or service point. When 'entity_id' or/and 'service_point_id' are provided, asset financial details are filtered by these parameters.
+
+.. table::
+
+   +-------------------------+--------------------------------------------------------------+---------------------------------------------------+-------------------------------------------------------+
+   | Field Name              | Data Type                                                    | Description                                       | Value                                                 |
+   +=========================+==============================================================+===================================================+=======================================================+
+   | asset_type_id           | int *(optional)*                                             | Asset Type ID - int                               | 1                                                     |
+   +-------------------------+--------------------------------------------------------------+---------------------------------------------------+-------------------------------------------------------+
+   | entity_id               | string *(optional)*                                          | Entity ID - UUID                                  | d666a904-5739-46c0-b70a-1cd57658a3f6                  |
+   +-------------------------+--------------------------------------------------------------+---------------------------------------------------+-------------------------------------------------------+
+   | service_point_id        | string *(optional)*                                          | Service Point Type ID - UUID                      | d666a904-5739-46c0-b70a-1cd57658a3f6                  |
+   +-------------------------+--------------------------------------------------------------+---------------------------------------------------+-------------------------------------------------------+
+
+Example Code
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    import requests
+
+    EVREKA360_API_BASE_URL = ""
+    ACCESS_TOKEN = ""
+
+    service_url = "/engagement/asset_financial_details"
+    headers = {
+        "Content-Type": "application/json; charset=utf-8", 
+        "Authorization": "Bearer " + ACCESS_TOKEN
+    }
+    
+    # filter example #1
+    # service_url += "?asset_type_id=1"
+    
+    # filter example #2
+    # service_url += "?entity_id=d666a904-5739-46c0-b70a-1cd57658a3f6"
+
+    # filter example #3 
+    # service_url += "?service_point_id=d666a904-5739-46c0-b70a-1cd57658a3f6"
+
+    # filter example #5 #To use multiple filters, use the & character between the filters.
```

Comment:
filter example 4,  instead of 5
most probably is copied from financial detail, can you fix it also